.gitignore

```diff
@@ -219,3 +219,4 @@ core/docker_data
 **/node/**
 **/node_modules/**
 /tweb/src/main/frontend/src/js/contracts/
+**/frontend/package-lock.json
```

Comment:
this should be committed
https://stackoverflow.com/questions/44206782/do-i-commit-the-package-lock-json-file-created-by-npm-5

Reply:
done